risc0/zkvm/platform/src/heap/mod.rs

```diff
@@ -13,7 +13,29 @@
 // limitations under the License.
 
 #[cfg(not(feature = "heap-embedded-alloc"))]
-mod bump;
+pub mod bump;
 
 #[cfg(feature = "heap-embedded-alloc")]
 pub mod embedded;
```

Comment:
Wondering why these are exposed at all, seems like a footgun and more API to maintain? Assuming `bump` needs to be exposed for the `init`, but just noting the expansion of the API (and breaking changes for a guest if you enable/disable the embedded alloc)

Reply:
Ah! Because I factored it one way, then changed it. Keeping it private is almost certainly better.
886d85bc32a66eb891f6a11e6261b1c341d8b6c4